lib/slim_fast/parser.ex

```diff
@@ -95,6 +71,40 @@ defmodule SlimFast.Parser do
     end
   end
 
+  @attr_regex ~r/([\w-]+)\s*=\s*(".+"|[\w-]+)/
```

Comment:
It does not support expression like `meta content=@user.name` and other simple elixir code =(